MSAL/src/native_auth/network/errors/MSALNativeAuthErrorMessage.swift

```diff
@@ -44,6 +43,7 @@ enum MSALNativeAuthErrorMessage {
     static let refreshTokenExpired = "Refresh token is expired"
     static let tokenNotFound = "Token not found"
     static let redirectUriNotSetWarning = "WARNING ⚠️: redirectUri not set during MSAL Native Auth initialization. Production apps must correctly configure a redirect URI and call acquireToken in response to all browserRequired errors. See https://learn.microsoft.com/entra/identity-platform/redirect-uris-ios"
+    static let unexpectedErrorBody = "Unexpected response body received"
```

Comment:
Maybe this should be changed to the more generic `unexpectedResponseBody` now that it is also used even when the problem is not with an error response?

Reply:
You're right, done.